html-forms/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+
+  <form>
+    <div>Email</div>
+    <input type="text" name="email"/>
```

Comment:
This input needs an `id` to correlate to the `for` attribute on it's corresponding `label` element